src/templates/template-vision-dcgan/main.py

```diff
@@ -25,8 +33,16 @@ def run(local_rank: int, config: Any):
     # create output folder and copy config file to output dir
     config.output_dir = setup_output_dir(config, rank)
     if rank == 0:
+        #::: if ((it.argparser == 'fire')) { :::#
```

Comment:
Same here as in https://github.com/pytorch-ignite/code-generator/pull/279/files#r1293734077